py-polars/polars/functions/range/date_range.py

```diff
@@ -184,11 +181,11 @@ def date_range(
         1985-01-09
     ]
 
-    Combine with :meth:`Expr.dt.month_end` to get the last day of the month:
+    Multiples of `interval` are added to the start date, so if you start on the
+    last day of the month and use "1mo" as the interval, each element will be the
+    end of its month:
```

Comment:
This is not true, right? If I start on `2022-05-30`, the next element will be `2022-06-30` which is not the last day of the month.

Reply:
you're right - maybe it's best to just remove this example altogether, I think it now does what people would expect it to